Examples/Cxx/Circuits/SP_SynGen3OrderVBR_SMIB_Fault.cpp

```diff
@@ -0,0 +1,208 @@
+#include <DPsim.h>
+#include "../Examples.h"
+
+using namespace DPsim;
+using namespace CPS;
+using namespace CPS::CIM;
+
+// ----- PARAMETRIZATION -----
+// General grid parameters
+Real nomPower = 555e6;
+Real VnomMV = 24e3;
+Real VnomHV = 230e3;
+Real nomFreq = 60;
+Real ratio = VnomMV/VnomHV;
+Real nomOmega= nomFreq * 2 * PI;
+
+//-----------Generator-----------//
+Examples::Components::SynchronousGeneratorKundur::MachineParameters syngenKundur;
+Real setPointActivePower=300e6;
+Real setPointVoltage=1.05*VnomMV;
+
+// HV line parameters referred to MV side
+Examples::Grids::CIGREHVAmerican::LineParameters lineCIGREHV;
+Real lineLength = 100;
+Real lineResistance = lineCIGREHV.lineResistancePerKm * lineLength * std::pow(ratio,2);
+Real lineInductance = lineCIGREHV.lineReactancePerKm * lineLength * std::pow(ratio,2) / nomOmega;
+Real lineCapacitance = lineCIGREHV.lineSusceptancePerKm * lineLength / std::pow(ratio,2) / nomOmega;
+Real lineConductance = 1e-15;
```

Comment:
Why did you choose here `1e-15` instead of `8e-2`? Maybe it's better practice to add the alternative value for the conductance parameter as well to `Examples::Grids::CIGREHVAmerican::LineParameters`, to place a comment there what its purpose is and use this alternative parameter (e.g. named `lineConductancePerKmAlternative`) then everywhere in order to harmonise